README.md

```diff
@@ -90,3 +90,44 @@ await mongoose.connect(url, defaultMongoOpts)
 const person = await PersonRepo.byID(id));
 const people = await PersonRepo.all({});
 ```
+
+## How to run a migration
+
+```shell
+yarn migrations init
+```
+
+The above command will initialize a `migrate-mongo-config.js` in the root directory and create a migration folder in your `src` folder.
+
+Now run
+
+```shell
+yarn migrations create [description]
+```
+
+The command helps to create a migration.ts file in the `src -> migration` directory
+
+Now it's time to write your migration code in the `function up() inside` `-migration.ts`
+
+```ts
+import { Db, MongoClient } from "mongodb";
+
+export async function up(db: Db, conn: MongoClient) {
+  // TODO write your migration here.
+  // Example:
+  // await db.collection('albums').updateOne({artist: 'The Beatles'}, {$set: {blacklisted: true}});
+}
+
+export async function down(db: Db, conn: MongoClient) {
+  // TODO write the statements to rollback your migration (if possible)
+  // Example:
+  // await db.collection('albums').updateOne({artist: 'The Beatles'}, {$set: {blacklisted: false}});
+}
+```
+
+In your index.ts add the code below, this helps to call the `function up()`
```

Comment:
We've never used it, but just in case, you could add notes on how the `down` function is meant to undo what your migration does.